load-after-keplr.html

```diff
@@ -0,0 +1,20 @@
+<script type="module">
+  const MAX_ATTEMPTS = 10;
+  const WAIT_PER_ATTEMPT = 10;
+  let attempts = 0;
+  const startWhenKeplrAvailable = () => {
+    console.debug('looking for keplr');
+    attempts += 1;
+    if (window.keplr) {
+      console.log('keplr found, loading app');
+      //IMPORT
```

Comment:
For clarity consider making the bundle url a const and have the replace the fake string literal with the correct one. Ie all the logic is here and the plug-in is just setting a config value.

Reply:
Closing this PR, but just FYI I did this because using a fake string literal got vite to complain statically that it couldn't find the module anywhere, I guess it checks the template before the transform happens.

Reply:
It complained statically even if it was `import(someValue)`?

Reply:
`import(someValue)` if `someValue` is an undefined variable makes linter complain. `import('someValue')` makes vite complain